### PR TITLE
added some file structure

### PR DESCRIPTION
**Pull request by hiddenkirby**
just to show you how github works :-p   i have my own fork of bdcityplanner .. i make changes and propose them to you like this..  you can comment on them.. wait till i fix the things you commented on .. or flat out decline

or accept. :)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<html>
+<head><title></title>
+<body>
+    <!-- libs -->
+    <script type="text/javascript" src="libs/md5.min.js"></script>
```
Comment (hiddenkirby) [inline index.html:5]: notice libs go above any app js files

File: javascripts/app.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+
+var foo = 'welcome';
+
+var hash = md5("value");
+alert(hash);
```
Comment (hiddenkirby) [inline javascripts/app.js:5]: here is where i hath md5'd the welcome msg ... for your ... pleasssuuurre..

File: index.html (hunk 1 above)
Comment (SvennEthir) [inline index.html:5]: No!

Comment (hiddenkirby) [inline index.html:5]: but for why?

Comment (SvennEthir) [inline index.html:5]: Because, reasons.

**Pull request closed without merging**